### DISCUSSION TITLE
App Crashing over and over on android 8.1.0

### DISCUSSION THREAD

**Discussion by droidguyg1** [osmandapp/OsmAnd#12564] [Q&A]
I've been using osmand for many years now.  And i bake it A LOT.  But recently, it's been crashing repeatedly over and over so often that more and more often, I have to give up and open Google maps instead just to get navigation going.  I have let it report the crash multiple time and added appropriate review in Google play but I wonder what is going on.  May be, it's just something on my phone?  Is anyone else. experiencing this problem?  It crashes often when I ask it to calculate a route.   But also, it happens sometimes when I just  browse the map.

**Comment by vshcherb** [comment]
Any crashes that you can attach here?

**Comment by droidguyg1** [comment]
Hi,

I have sent multiple crashes directly from the osmand.  However, when it
finally stopped working completely, i have reinstalled osmand from scratch
on my phone and it's working fine.  That previous installation worked for
about two years.  Maybe, it accumulated some craft?

Stan Berka
